test/libevmasm/Optimiser.cpp

```diff
@@ -920,8 +920,8 @@ BOOST_AUTO_TEST_CASE(block_deduplicator_loops)
 		AssemblyItem(PushTag, 2),
 		Instruction::JUMP,
 	};
-	BlockDeduplicator dedup(input);
-	dedup.deduplicate();
+	BlockDeduplicator deduplicator(input);
+	deduplicator.deduplicate();
```

Comment:
Slightly surprised that this is needed. The spell check checks the spelling of variables?

Reply:
Apparently yes.